pyproject.toml

```diff
@@ -1,54 +1,60 @@
 [project]
 name = "qpc"
 version = "1.13.0"
-license = {file = "LICENSE"}
+description = "The command-line client for quipucords servers"
+authors = [{ name = "QPC Team", email = "quipucords@redhat.com" }]
 requires-python = ">=3.12,<3.14"
-dynamic = ["scripts"]
+license = { file = "LICENSE" }
+classifiers = ["License :: OSI Approved :: GNU General Public License v3 (GPLv3)"]
+dependencies = [
+    "requests>=2.28.1",
+    "cryptography>=37.0.4",
+    "setuptools>=70.0.0,<71",
+    "faker>=20.0.3,<21",
+]
+#dynamic = ["scripts"]
 
-[tool.poetry]
-name = "qpc"
-version = "1.13.0"
-description = ""
-classifiers=[
-    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+[project.scripts]
+qpc = "qpc.__main__:main"
+
+[dependency-groups]
+dev = [
+    "coverage>=6.4.2",
+    "pytest>=7.1.2,<8.0.0",
+    "pytest-cov>=3.0.0",
+    "pytest-lazy-fixture>=0.6.3",
+    "requests-mock>=1.9.3",
+    "pytest-mock>=3.8.2,<4",
+    "rstcheck>=6.1.1,<7",
+    "ruff>=0,<1",
+    "pip-tools>=7.1.0,<8",
+    "pybuild-deps>=0.1.1,<0.2",
 ]
-authors = ["QPC Team <quipucords@redhat.com>"]
+build = ["sphinx>=7.2.6,<8"]
+
+[tool.uv]
+default-groups = [
+    "dev",
+    "build",
+]
```

Comment:
maybe add `required-version = ">=0.6.11,<0.7"` for consistency with main quipucords.

Reply:
Good catch! Added in a fixup commit that I will squash assuming CI checks are still happy.